pkgs/data/fonts/whatsapp-emoji/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, imagemagick
+, nix-update-script
+, pngquant
+, python3Packages
+, which
+, zopfli
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "whatsapp-emoji-linux";
+  version = "2.22.8.79-1";
+
+  src = fetchFromGitHub {
+    rev = "refs/tags/2.22.8.79-1";
+    owner = "dmlls";
+    repo = pname;
+    sha256 = "sha256-AYdyNZYskBNT3v2wl+M0BAYi5piwmrVIDfucSZ3nfTE=";
+  };
+
+  makeFlags = [
+    "PREFIX=$(out)"
+  ];
```

Comment:
AFAIK this is the default.

Reply:
if i don't specify this it will try to write at $HOME

https://github.com/dmlls/whatsapp-emoji-linux/blob/8bfc015ad7085a2ef515f5e95147765a7d7d3767/Makefile#L29

Reply:
> AFAIK this is the default.

This is not enforced outside `configurePhase`. Indeed you can deadgrep the Nixpkgs tree and find this.

Reply:
without it it doesnt work

setting PREFIX is much easier than do some symlink shenanigans to redirect the file so closing comment